OpenRA.Mods.Common/Traits/ParaDrop.cs

```diff
@@ -70,30 +83,38 @@ void ITick.Tick(Actor self)
 				OnExitedDropRange(self);
 
 			// Are we able to drop the next trooper?
-			if (!inDropRange || cargo.IsEmpty(self))
+			if (!inDropRange || cargo.IsEmpty(self) || !self.World.Map.Contains(self.Location))
 				return;
 
-			if (droppedAt.Contains(self.Location) || (checkForSuitableCell && !IsSuitableCell(cargo.Peek(self), self.Location)))
-				return;
+			var dropActor = cargo.Peek(self);
+			var dropPositionable = dropActor.TraitOrDefault<IPositionable>();
+			var dropCell = self.Location;
+			var dropSubCell = dropPositionable.GetAvailableSubCell(dropCell);
```

Comment:
Won't this crash if `dropPositionable` is null? In which case you might as well drop the `OrDefault` above.